tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -824,5 +830,34 @@ int main(int Ac, char **Av) {
                 << ", size in bytes = " << SpecConst.Size
                 << ", type = " << SpecConst.Type << "\n";
   }
+
+  if (SPIRVPrintReport) {
```

Comment:
For the human-readable form of the report, we should be more user-friendly, I think. I mean that all binary fields should be decoded (version, capabilities, etc.)

Reply:
Probably the human-readable form of the report deserves even a separate (additional) API call that takes SPIRV::SPIRVModuleReport as an argument and returns all binary fields decoded.